pmm-app/src/pmm-add-instance-app-panel/components/AddInstance/AddInstance.constants.ts

```diff
@@ -0,0 +1,9 @@
+import { Messages } from './AddInstance.messages';
```

Comment:
Can we rename the panel directory to `pmm-add-instance` to match the rest?

Reply:
done